src/emulator/config/include/config/config_func.h

```diff
@@ -17,20 +17,20 @@
 
 #pragma once
 
-#include <host/config.h>
+#include <config/config.h>
 #include <util/exit_code.h>
 #include <util/fs.h>
 
 class Root;
 
-namespace app {
+namespace config {
```

Comment:
This is a standalone module now, so rename this file to functions.h (to synchronize with other modules)